config/locales/en.yml

```
@@ -250,7 +250,7 @@ en:
           year_label: Year label
     statutory_representative:
       inform:
-        body: Body
+        body: Bonjour <br> Salut
```

Are we sure about that line?
```suggestion
        body: Body
```